api/controllers/v1/massif/get-statistics.js

```diff
@@ -0,0 +1,79 @@
+const ControllerService = require('../../../services/ControllerService');
+const DashboardDataMassifService = require('../../../services/StatisticsMassifService');
+const ErrorService = require('../../../services/ErrorService');
+
+module.exports = async (req, res) => {
+  try {
+    const params = {
+      controllerMethod: 'MassifController.get-statistics',
+    };
+
+    if (!req.params.id) {
+      return ControllerService.treat(
+        req,
+        'Missing massif id',
+        null,
+        params,
+        res
+      );
+    }
+
+    await DashboardDataMassifService.createMaterializedView();
```

Comment:
This need to go away and be replaced by a cron refresh + a  .sql file a to create the view 1 time only (see the README of the sql directory)

Reply:
Yes we know, we focused on other features for now but the cron management is planned.

Reply:
So can you please remove this from the PR to prevent a bad line to be introduced to the source code ?

Reply:
Je valide et marge cette PR mais il faudra absolument enlever cette ligne `await DashboardDataMassifService.createMaterializedView();` qui n'a rien à faire dans un route d'API...
la vue est créé une bonne fois pour toute et ce sera un service externe qui la mettre à jour ;)